# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/PSocketProcess.ipdl
# Commit: c684f79d8c67
# Full Hash: c684f79d8c673f21179fce00358d70845a9692c0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Rename PWebrtcProxyChannel to PWebrtcTCPSocket. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45098
# ==============================================================================

diff -r 82af7b115f60 -r c684f79d8c67 netwerk/ipc/PSocketProcess.ipdl
--- a/netwerk/ipc/PSocketProcess.ipdl	Tue Sep 17 18:15:23 2019 +0000
+++ b/netwerk/ipc/PSocketProcess.ipdl	Tue Sep 17 18:15:41 2019 +0000
@@ -7,7 +7,7 @@
 include protocol PSocketProcessBridge;
 include protocol PProfiler;
 include protocol PProxyConfigLookup;
-include protocol PWebrtcProxyChannel;
+include protocol PWebrtcTCPSocket;
 
 include MemoryReportTypes;
 include PBrowserOrId;
@@ -29,8 +29,8 @@
 protocol PSocketProcess
 {
   manages PDNSRequest;
+  manages PWebrtcTCPSocket;
   manages PProxyConfigLookup;
-  manages PWebrtcProxyChannel;
 
 parent:
   async InitCrashReporter(Shmem shmem, NativeThreadId threadId);
@@ -44,7 +44,7 @@
   async RecordChildEvents(ChildEventData[] events);
   async RecordDiscardedData(DiscardedData data);
 
-  async PWebrtcProxyChannel(TabId tabId);
+  async PWebrtcTCPSocket(TabId tabId);
   async PDNSRequest(nsCString hostName, OriginAttributes originAttributes,
                     uint32_t flags);
   async PProxyConfigLookup();